warp10/src/main/java/io/warp10/script/WarpScriptStack.java

```diff
@@ -122,6 +122,11 @@ public static enum Signal {
    */
   public static final String ATTRIBUTE_INFOMODE = "infomode";
 
+  /**
+   * List of parsing errors generated in WarpScript audit mode
+   */
+  public static final String ATTRIBUTE_PARSING_ERRORS = "wsaudit.errors";
```

Comment:
If I understood correctly your PR, the value of this attribute is a list of WarpScriptTraceableStatement (to be renamed), not a list of errors, so renaming it is advised.

Reply:
It is a list of WarpScriptAuditStatement that contains errors only (unknown functions, or parsing exceptions).
It avoids to walk through nested macros to look for these errors... So I named it errors.
I changed the comment to explain.